Rakefile

```diff
@@ -5,11 +5,16 @@ require "bundler"
 Bundler::GemHelper.install_tasks
 
 desc "Run basic tests"
-Rake::TestTask.new("test") { |t|
+Rake::TestTask.new("test") do |t|
   t.libs << "test"
   t.pattern = "test/*_test.rb"
   t.verbose = true
   t.warning = true
-}
+end
 
-task default: [:test]
+desc "Lint Ruby"
+task :lint do
+  sh "bundle exec rubocop --format clang"
+end
```

Comment:
We can use the built in rubocop task rather than defining our own. 

So this could have a require earlier of `require 'rubocop/rake_task'` and initialise `RuboCop::RakeTask.new` in this file then the default task can be `task default: %i[test rubocop]` without needing a task defined.